package-lock.json

```diff
@@ -3283,27 +3283,27 @@
       "dependencies": {
```

Comment:
You didn't install a new package so this file shouldnt be changing

Reply:
I had a merge conflict, I had to rebase and run npm install.